CONTRIBUTING.md

```diff
@@ -17,24 +17,53 @@ In addition to the above, the icons must have an outlined (not filled) style. If
 ## Adding an icon to Lawnicons
 Here’s how to add an icon to&nbsp;Lawnicons:
 
-1. Prepare your icon in the SVG format, adhering to the [above guidelines](#icon-guidelines). Use snake case for the filename (e.g.&nbsp;`files_by_google.svg`).
+1. Prepare your icon in the SVG format, adhering to the [above guidelines](#icon-guidelines). Use snake case for the filename (e.g.,&nbsp;`youtube_music.svg`).
 
 1. Add the ready SVG to the `svgs`&nbsp;directory.
 
-1. Using Android Studio, convert the SVG to an XML drawable, and add the XML drawable to the `app/src/main/res/drawable` directory. Use snake case for the drawable name (e.g. `files_by_google`). You can keep all settings at their&nbsp;defaults.
 
-    ![](./contributing-image-3.png) ![](./contributing-image-4.png)
-
-1. Add a new line to `app/src/main/res/xml/grayscale_icon_map.xml` (sorted alphabetically by drawable name), and map the new icon to a package name and app name. For&nbsp;example:
+1. Add a new line to `app/assets/appfilter.xml` (in alphabetical order), and map the new icon to a package name and app's activity. For&nbsp;example:
 
     ```xml
-    <icon drawable="@drawable/files_by_google" package="com.google.android.apps.nbu.files" name="Files by Google" />
+    <item component="ComponentInfo{app.lawnchair.lawnicons/app.lawnchair.lawnicons.MainActivity}" drawable="lawnicons"/> 
```

Comment:
I'd recommend keeping the same app/package name as before (Files by Google) for consistency with the rest of the steps/images.

This also includes the bottom section.